polkadot/xcm/docs/src/cookbook/other_parachain_tokens/network.rs

```diff
@@ -0,0 +1,150 @@
+// Copyright Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Mock network
+
+use frame::deps::{
+	frame_system,
+	sp_io::TestExternalities,
+	sp_runtime::{AccountId32, BuildStorage},
+};
```

Comment:
I re-export the `TestExternalities` as `TestState` in `testing_prelude::*`. @gupnik I think this is one opinion that we should revise, it causes too much needless breaking change. Let's re-export both `TestExternalities::{self, self as TestState}`?

Reply:
Otherwise I generally think you can replace all of these lines with `frame::testing_prelude::*`.

Reply:
I managed to remove all the lines except for `sp_runtime::AccountId32`